build_tools/docker/base/Dockerfile

```
@@ -130,6 +130,26 @@ ENV PYTHON_BIN /usr/bin/python3
 
 ##############
 
+
+######## IREE ROCM DEPS ########
+ARG ROCM_VERSION=latest
```

The other sections start by setting the WORKDIR so that any intermediate files are isolated. I don't think what you have here is going to create any of those, but I think it would be good to follow the same pattern

```suggestion
WORKDIR /install-rocm
ARG ROCM_VERSION=latest
```

---

done.